pkg/clusterversion/cv.go

```diff
@@ -199,8 +239,13 @@ func GetCurrentVersion(clusterVersion *configv1.ClusterVersion) (string, error)
 	}
 
 	if len(gotVersion) == 0 {
-		return gotVersion, fmt.Errorf("Failed to get current version")
+		return gotVersion, fmt.Errorf("failed to get current version")
 	}
 
 	return gotVersion, nil
 }
+
+// empty function checks if a given string is empty or not.
+func empty(s string) bool {
```

Comment:
do we need a separated func to check the string empty? can be done by `if len(string) == 0`

Reply:
Yes, but based on previous review comments like https://github.com/openshift/managed-upgrade-operator/pull/253#discussion_r659426643, the code seemed more readable so thought of putting it in empty(). A part of making this better would also be taken up in https://issues.redhat.com/browse/OSD-7686. Let me know what you think.